src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-unit-form/attachment-unit-form.component.ts

```diff
@@ -83,6 +85,7 @@ export class AttachmentUnitFormComponent implements OnInit, OnChanges {
             releaseDate: [undefined as dayjs.Dayjs | undefined],
             version: [1],
             updateNotificationText: [undefined as string | undefined, [Validators.maxLength(1000)]],
+            learningGoals: [undefined as LearningGoal[] | undefined],
```

Comment:
How about creating a separate type definition of this group instead of casting `undefined` into the wanted types

Reply:
I don't think this makes sense for this very specific use case in the Angular Forms definition.